src/annotator/anchoring/text-range.js

```diff
@@ -1,3 +1,21 @@
+/**
+ * Return the combined length of text nodes contained in `node`.
+ *
+ * This is different than `node.textContent` if called on a comment or processing
```

Comment:
It might be useful to expand this comment to explain the difference, as in the commit message. I know it's in the commit message, but that might not get seen by an active dev in the code. Might be as simple as "don't count text in nodes unless they're Element or Text nodes" (that's more explanative than "this is different").